packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -116,11 +116,11 @@
       + .label {
         .box {
           @include outline(
-            var(--ColorFocusOutlineInvalid),
+            var(--InvalidFocusOutlineColor),
             var(--FocusOutlineWidth)
           );
```

Comment:
would be good to see this mixin removed, it only contains two lines of CSS so its a minimal change while we're here